forui/lib/src/widgets/tile/tile_content.dart

```diff
@@ -0,0 +1,524 @@
+    final FTileStyle(:contentStyle, :dividerStyle) = tileStyle;
+
+    Widget tile = Padding(
+      padding: contentStyle.padding,
+      child: Row(
+        children: [
+          if (prefixIcon case final prefixIcon?)
+            Padding(
+              padding: EdgeInsets.only(right: contentStyle.prefixIconSpacing),
+              child: FIconStyleData(
+                style: style.prefixIconStyle,
+                child: prefixIcon,
+              ),
+            ),
+          Column(
+            mainAxisSize: MainAxisSize.min,
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              merge(
+                style: style.titleTextStyle,
+                textHeightBehavior: const TextHeightBehavior(
+                  applyHeightToFirstAscent: false,
+                  applyHeightToLastDescent: false,
+                ),
+                overflow: TextOverflow.ellipsis,
+                child: title,
+              ),
+              if (subtitle case final subtitle?)
+                Padding(
+                  padding: EdgeInsets.only(top: contentStyle.titleSpacing),
+                  child: merge(
+                    style: style.subtitleTextStyle,
+                    textHeightBehavior: const TextHeightBehavior(
+                      applyHeightToFirstAscent: false,
+                      applyHeightToLastDescent: false,
+                    ),
+                    overflow: TextOverflow.ellipsis,
+                    child: subtitle,
+                  ),
+                ),
+            ],
+          ),
+          const Spacer(),
+          if (details case final details?)
+            merge(
+              style: style.detailsTextStyle,
+              textHeightBehavior: const TextHeightBehavior(
+                applyHeightToFirstAscent: false,
+                applyHeightToLastDescent: false,
+              ),
+              overflow: TextOverflow.ellipsis,
+              child: details,
+            ),
+          if (suffixIcon case final suffixIcon?)
+            Padding(
+              padding: EdgeInsets.only(left: contentStyle.suffixIconSpacing),
+              child: FIconStyleData(
+                style: style.suffixIconStyle,
+                child: suffixIcon,
+              ),
+            ),
+        ],
+      ),
+    );
+
+    if (divider == FTileDivider.full && index < length - 1) {
+      tile = Column(
+        mainAxisSize: MainAxisSize.min,
+        children: [tile, FDivider(style: dividerStyle)],
+      );
+    }
+
+    return tile;
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties
+      ..add(DiagnosticsProperty('style', style))
+      ..add(ColorProperty('background', background));
+  }
+}
+
+class _TitleAlignedContent extends StatelessWidget {
```

Comment:
What's the difference between this and _Content? From a glance they seem to have the same fields :(

Reply:
_TitleAlignedContent introduces a column which adds a divider below the row that contains the title, subtitle, details and suffix icon.

Reply:
Based on what you mentioned, it add a divider between Tiles? In any case, I don't quite understand why this is named `TitleAligned` though. It doesn't quite seem to do with anything with the alignment but more of the fact that there is an additional divider.